reframe/schemas/config.json

```
@@ -274,6 +274,7 @@
                 "type": "object",
                 "properties": {
                     "name": {"type": "string"},
+		    "emit_num_nodes": {"type": "boolean"},
```

```suggestion
                    "emit_num_nodes": {"type": "boolean"},
```
You should use spaces instead of tabs.